CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -37,1064 +37,1065 @@
         <note>Shown in Settings</note>
       </trans-unit>
       <trans-unit id="For new &amp; popular media." xml:space="preserve">
-        <source>For new &amp; popular media.</source>
+        <source>For new &amp; popular media.</source><target>Для нових та популярних медія.</target>
         <note>Shown in Settings</note>
       </trans-unit>
       <trans-unit id="Free Trial" xml:space="preserve">
-        <source>Free Trial</source>
+        <source>Free Trial</source><target>Безкоштовний пробний період</target>
         <note/>
       </trans-unit>
       <trans-unit id="Free searches remaining" xml:space="preserve">
-        <source>Free searches remaining</source>
+        <source>Free searches remaining</source><target>Залишилось безкоштовних пошуків</target>
         <note/>
       </trans-unit>
       <trans-unit id="Genre" xml:space="preserve">
-        <source>Genre</source>
+        <source>Genre</source><target>Жанр</target>
         <note/>
       </trans-unit>
       <trans-unit id="Genres" xml:space="preserve">
-        <source>Genres</source>
+        <source>Genres</source><target>Жанри</target>
         <note/>
       </trans-unit>
       <trans-unit id="Guests" xml:space="preserve">
-        <source>Guests</source>
+        <source>Guests</source><target>Гості</target>
         <note/>
       </trans-unit>
       <trans-unit id="Hand your phone to someone knowing they **cannot mess up your photos**." xml:space="preserve">
-        <source>Hand your phone to someone knowing they **cannot mess up your photos**.</source>
+        <source>Hand your phone to someone knowing they **cannot mess up your photos**.</source><target>Передавайте свій телефон, знаючи що вони **не зможуть зіпсувати ваші фото**.</target>
         <note>Peek-a-View subtitle in Settings</note>
       </trans-unit>
       <trans-unit id="Height" xml:space="preserve">
-        <source>Height</source>
+        <source>Height</source><target>Зріст</target>
         <note/>
       </trans-unit>
       <trans-unit id="Hide Spoilers" xml:space="preserve">
-        <source>Hide Spoilers</source>
+        <source>Hide Spoilers</source><target>Приховати спойлери</target>
         <note/>
       </trans-unit>
       <trans-unit id="Hide Spoilers…" xml:space="preserve">
-        <source>Hide Spoilers…</source>
+        <source>Hide Spoilers…</source><target>Приховати спойлери…</target>
         <note/>
       </trans-unit>
       <trans-unit id="How seasons and episodes are sorted." xml:space="preserve">
-        <source>How seasons and episodes are sorted.</source>
+        <source>How seasons and episodes are sorted.</source><target>У якому порядку відображати сезони та епізоди.</target>
         <note/>
       </trans-unit>
       <trans-unit id="I'm in a popover" xml:space="preserve">
-        <source>I'm in a popover</source>
+        <source>I'm in a popover</source><target>Я у віконці</target>
         <note>Do not translate</note>
```

Comment:
Do not translate

Reply:
https://github.com/cliss/callsheet-localizations/pull/3/commits/92d8298c05e7a90ebba434bb90661a256defdab3